### ISSUE TITLE
1.1. figure with schematic going from spectrogram --> annotation

### ISSUE THREAD

**Issue by NickleDave** [yardencsGitHub/tweetynet#57]
> 1.1. The architecture is clearly laid out in Fig 4 but a much better job of showing visualizations of how an example spectrogram is annotated will be helpful. For example, grayscale matrices (as in the bottom panels of Fig. 8) could appear earlier in the paper with bengalese finch examples to really show readers how each ~2.7 ms timebin (frame) is being assessed, and how each frame's label is converted into a gap-annotated syllable-gap sequence. I'd like to see some kind of visualization that brings the reader from a syllable in a spectogram to the matrix, to the argmax function through to annotation will be helpful from the outset. The schematic could, if possible, clarify how the LSTM is operating over frames including how many it holds in memory.

visualization that brings the reader from a syllable in a spectogram to the matrix, to the argmax function through to annotation will be helpful 

- [ ] show spectrogram -> window -> network ->  frame labels -> annotated sequence, including majority vote
- [ ] schematic should clarify how the LSTM is operating over frames
  + would be helpful to show we use the concatenate approach to combining biLSTM outputs, and use **all** hidden states

**Comment by NickleDave** [comment]
"For example, grayscale matrices (as in the bottom panels of Fig. 8) could appear earlier in the paper with bengalese finch examples to really show readers how each ~2.7 ms timebin (frame) is being assessed, and how each frame's label is converted into a gap-annotated syllable-gap sequence."

I have a branch in progress that will make this simple
https://github.com/NickleDave/vak/tree/add-save-net-outputs-%2390

**Comment by NickleDave** [comment]
- [ ] show e.g. one second window of BF song, with few syllables
- [ ] include output matrix, logits or softmax
- [ ] highlight argmax

**Comment by NickleDave** [comment]
discussing with @yardencsGitHub now
- [ ] have softmax matrix with red square around argmax
- [ ] then have segments before and after majority vote below that
- [ ] consider rotating schematic so flow is top-to-bottom -- figure is already pretty wide as it is

**Comment by NickleDave** [comment]
Fixed in #113 (plus or minus slight tweaking)
